frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineRunItem.tsx

```diff
@@ -31,7 +32,11 @@ const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
       <Grid hasGutter>
```

Comment:
The `li` one line up could use with the `odc-pipeline-run-item` class too to maintain a good structure.